Library/Formula/fltk.rb

```diff
@@ -11,27 +9,42 @@ class Fltk < Formula
 
   option :universal
 
+  stable do
+    # Fltk 1.3.4 include support for El Capitan. Remove on update.
```

Comment:
These two lines (The comment and the depends_on) should be one empty line beneath the `:DATA` patch line.